src/domain/relay/limit-addresses.mapper.spec.ts

```diff
@@ -66,7 +66,7 @@ describe('LimitAddressesMapper', () => {
       const safe = safeBuilder().build();
       const safeAddress = getAddress(safe.address);
       const data = execTransactionEncoder()
-        .with('to', getAddress(faker.finance.ethereumAddress()))
+        .with('value', faker.number.bigInt())
```

Comment:
This mock was incorrect. A `value` should be being sent but the builder defaults to `0`.